fortran/fortran-polyroots/Portfile

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           fortran 1.0
+
+github.setup        jacobwilliams polyroots-fortran ef5433e943eeb3edec7f5fce62ad7a09412cb23d
```

Comment:
please use the released version as it is tagged. No tags please unless a -devel version as per the usual guidelines.